### PR TITLE
Inserted new user on the click of Register button

### PR DESCRIPTION
**Pull request by HarpreetKaurLNG**
Inserted new user on the click of Register button

### CODE REVIEW

File: MVCCore.BL/Register.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        public Register(Models.AppContext context)
+        {
+           _dbContext = context;
+        }
+
+        public async Task<int> SaveNewUser(RegisterModel register)
+        {
+                     _dbContext.Add(register);
+                     await _dbContext.SaveChangesAsync();
+            return intResponse =1 ;
```
Comment (VikasNischalDA) [inline MVCCore.BL/Register.cs]: why returning hard coded values

File: MVCCore.BL/Register.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        }
+
+        public async Task<int> CheckUserExistsOrNot(string email)
+        {
+              intResponse = await _dbContext.Register.Where(x => x.Email == email).Select(x => x.UserId).FirstOrDefaultAsync() ;
```
Comment (VikasNischalDA) [inline MVCCore.BL/Register.cs]: should it not be bool as per method name

File: MVCCore.Models/Login.cs [hunk 3]
```diff
@@ -19,9 +19,6 @@ public class Login
         [Display(Name = "Password")]
         [MaxLength(50)]
```
Comment (VikasNischalDA) [inline MVCCore.Models/Login.cs:20]: why there is data validation attribute

File: MVCCore.Models/RegisterModel.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace MVCCore.Models
+{
+   public class RegisterModel
+    {
+        [Key]
+        public Int32 UserId { get; set; }
+
+        [Display(Name = "User Name")]
+        [MaxLength(50)]
```
Comment (VikasNischalDA) [inline MVCCore.Models/RegisterModel.cs:15]: here also

Comment (HarpreetKaurLNG) [reply MVCCore.Models/RegisterModel.cs:15]: Which one only Display Name and max length is used

File: MVCCore.Models/Login.cs (hunk 3 above)
Comment (HarpreetKaurLNG) [reply MVCCore.Models/Login.cs:20]: Which one? only Display Name and max length is used

File: MVCCore.BL/Register.cs (hunk 2 above)
Comment (HarpreetKaurLNG) [reply MVCCore.BL/Register.cs]: Modification done

File: MVCCore.BL/Register.cs (hunk 1 above)
Comment (HarpreetKaurLNG) [reply MVCCore.BL/Register.cs]: It was for testing purpose, Modification done.

File: MVCCore.Models/Login.cs (hunk 3 above)
Comment (VikasNischalDA) [reply MVCCore.Models/Login.cs:20]: max length validation attribute

**Review by VikasNischalDA — APPROVED** [review]

**Pull request merged**